medical_emr/__openerp__.py

```diff
@@ -36,57 +37,10 @@
 
     'name': 'Medical EMR : Electronic Medical Record management for Medical',
     'version': '1.0',
```

Comment:
@lasley  @nhomar The rule is now part of the official guideline https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers. @lasley Can you modify the version number according to the guidelines?